config/crd/bases/clusters.instaclustr.com_postgresqls.yaml

```diff
@@ -183,6 +267,7 @@ spec:
               privateNetworkCluster:
                 type: boolean
               resizeSettings:
+                description: ClusterResourceRefs   *ClusterResourceRefs `json:"clusterResourceRefs,omitempty"`
```

Comment:
Looks pretty strange to me

Reply:
it was triggered by unnecessary  comment